site/cypress/e2e/Cypress-TAGrading/rubric_grading.spec.js

```diff
@@ -0,0 +1,61 @@
+describe('Test cases for TA grading page', () => {
+    it('Grader should be able to add and remove overall comments', () => {
+        cy.login('instructor');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'grade?who_id=jKjodWaRdEV9pBb&sort=id&direction=ASC']);
+        cy.get('body').type('{A}');
+        cy.get('body').type('{G}');
+        cy.get('[data-testid="grading-rubric"]').should('contain', 'Grading Rubric');
+        cy.get('[data-testid="component-container"]').its('length').should('eq', 4);
+        cy.get('[data-testid="component-container"]').eq(0).should('contain', 'Read Me');
+        cy.get('[data-testid="component-container"]').eq(1).should('contain', 'Coding Style');
+        cy.get('[data-testid="component-container"]').eq(2).should('contain', 'Documentation');
+        cy.get('[data-testid="component-container"]').eq(3).should('contain', 'Extra Credit');
+        cy.get('[data-testid="component-64"]').should('contain', 'Read Me');
+        cy.get('[data-testid="component-64"]').click();
+        cy.get('[data-testid="component-64"]')
+            .should('contain', 'Full Credit')
+            .should('contain', 'Minor errors in Read Me')
+            .should('contain', 'Major errors in Read Me or Read Me missing');
```

Comment:
```suggestion
        cy.get('[data-testid="component-64"]')
            .should('contain', 'Full Credit')
            .and('contain', 'Minor errors in Read Me')
            .and('contain', 'Major errors in Read Me or Read Me missing');
```

Same with other uses of chained should calls